pkg/util/winutil/winver.go

```diff
@@ -0,0 +1,127 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+// +build windows
```

Comment:
Could we add an empty line between our header and the build flag (here and in every other file) to follow what's done in the other packages in the agent ?